.circleci/config.yml

```diff
@@ -1,16 +1,27 @@
-orbs:
-  react: "thefrontside/react@dev:alpha"
-version: 2.1
-workflows:
-  push:
-    jobs:
-      - react/install
-      - react/eslint:
-          requires:
-            - react/install
-      - react/test:
-          requires:
-            - react/install
-      - react/coverage:
-          requires:
-            - react/install
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:10.15.2
+    steps:
+      - checkout
+      - restore_cache: # special step to restore the dependency cache
+          key: dependency-cache-{{ checksum "package.json" }}
+      - run:
+          name: Setup dependencies
+          command: npm install
```

Comment:
Consider using yarn package manager instead of npm as agreed by the team.

Reply:
I've not found a descriptive way of doing circle ci with yarn 
However, the only thing needed is the package.json and tests 
We can use npm to run tests on circle ci and use yarn in other places 

what do you think?